### DISCUSSION TITLE
Type definitions for ProBot extensions

### DISCUSSION THREAD

**Discussion by maximousblk** [DefinitelyTyped/DefinitelyTyped#55279] [Request a new @types package]
The probot extensions [`probot-commands`](http://github.com/probot/commands), [`probot-metadata`](http://github.com/probot/metadata) and [`probot-attachments`](http://github.com/probot/attachments) do not have types. Though the main [`probot`](https://github.com/probot/probot) package has types

**Comment by martin-badin** [comment]
I might it will be better to ask for types probot developers on their GitHub project.
